### PR TITLE
Algolia Package installation 

### PR DESCRIPTION
**Pull request by premalrupnur**
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/44)

### CODE REVIEW

**Comment by codecakes** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6303157&bid=20497935)

File: covidX/settings.py [hunk 1]
```diff
@@ -183,3 +184,8 @@
     "SCHEMA_OUTPUT": "schema.json",
     "SCHEMA_INDENT": 2,
 }
+ALGOLIA = {
+    'APPLICATION_ID': 'YourApplicationID',
+    'API_KEY': 'YourAdminAPIKey'
+}
+
```
Comment (codecakes) [inline covidX/settings.py:191]: Codacy found an issue: [Trailing newlines](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6303157)

File: covidX/settings.py [hunk 2]
```diff
@@ -183,3 +184,4 @@
     "SCHEMA_OUTPUT": "schema.json",
     "SCHEMA_INDENT": 2,
 }
+ALGOLIA = {"APPLICATION_ID": "YourApplicationID", "API_KEY": "YourAdminAPIKey"}
```
Comment (codecakes) [inline covidX/settings.py:187]: @premalrupnur write an `os.getenv` to get `ALGOLIA_APPLICATION_ID` and `ALGOLIA_API_KEY` ;
from env vars. then you can do:
`ALGOLIA = {"APPLICATION_ID": ALGOLIA_APPLICATION_ID, "API_KEY": ALGOLIA_API_KEY}`

**Review by codecakes — CHANGES_REQUESTED** [review]
looks good overall. make the necessary changes and ill approve.

File: covidX/settings.py (hunk 2 above)
Comment (premalrupnur) [reply covidX/settings.py:187]: ok will do

Comment (premalrupnur) [reply covidX/settings.py:187]: I have to replace the "ALGOLIA_APPLICATION_ID" with the value returned after using getenv method ,right

**Comment by premalrupnur** [comment]
After you find this issue is resolved please assign some other  issues(preferably introductory) also , would be glad to work on it

File: covidX/settings.py (hunk 2 above)
Comment (codecakes) [reply covidX/settings.py]: Yes, ALGOLIA_APPLICATION_ID is a const

**Comment by premalrupnur** [comment]
Hey what should I  do further

**Comment by heidar55** [comment]
Can I participate on your project?

File: requirements.txt [hunk 3]
```diff
@@ -93,6 +93,7 @@ wasmer==0.4.1             # via -r requirements.txt, fastdiff
 wcwidth==0.2.5            # via -r requirements.txt, prompt-toolkit
 werkzeug==1.0.1           # via -r requirements.txt
 xlrd==1.2.0               # via -r requirements.txt
+algoliasearch-django==1.7.1  # via -r requirements.txt
```
Comment (codecakes) [inline requirements.txt:96]: In a virtualenv run `pip-compile --allow-unsafe --generate-hashes requirements.txt` ;
make sure additionally u r able to install using pip3 in that env.

**Review by codecakes — CHANGES_REQUESTED** [review]
ive added some comments. take a fresh merge from master first into your branch

**Comment by premalrupnur** [comment]
> ive added some comments. take a fresh merge from master first into your branch

Ok will do

**Comment by premalrupnur** [comment]
Hey I got confused with source control and have done mistakes ig I'll close this pr and create a fresh pr

**Comment by codecakes** [comment]
TODO(@codecakes ): add to secrets mgr and gh wf Algolia API;

**Review by codecakes — CHANGES_REQUESTED** [review]
In the secrets.sh file add this line:
```bash
echo "ALGOLIA_API_KEY=$(gcloud secrets versions access latest --secret='ALGOLIA_API_KEY')" >> .env
echo "ALGOLIA_APPLICATION_ID=$(gcloud secrets versions access latest --secret='ALGOLIA_APPLICATION_ID')" >> .env
```

**Comment by premalrupnur** [comment]
> In the secrets.sh file add this line:
> 
> ```shell
> echo "ALGOLIA_API_KEY=$(gcloud secrets versions access latest --secret='ALGOLIA_API_KEY')" >> .env
> echo "ALGOLIA_APPLICATION_ID=$(gcloud secrets versions access latest --secret='ALGOLIA_APPLICATION_ID')" >> .env
> ```

May I know why this particular snippet needs to be added I mean what it does

**Review by codecakes — APPROVED** [review]
@premalrupnur merge this

**Comment by codecakes** [comment]
> Can I participate on your project?

@heidar55 ofcourse. head to https://forum.mycovidconnect.com/d/3-start-here-introduce-yourself-getting-to-know-the-project to get started

**Pull request merged**